docs/CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ which allows both creating discussion, ideas, polls and more as well as voting o
 * For more in-depth technical discussion you are welcome to join the #dev-chat channel in
   the [Deluge User Discord](https://discord.com/invite/gWaJEh6pPP).
 
-# Code review and testing
+## Code review and testing
```

Comment:
This section feels a bit out of place. It feels like it should have come after the Repository contributions section which was removed.

I would keep a header Repository contributions and then refer dev contributors to the dev documentation

Reply:
Changed header and added a distinct link to 'Guidelines...' before the 'Additional Developer Resources' footer.

Reply:
> Changed header and added a distinct link to 'Guidelines...' before the 'Additional Developer Resources' footer.

Hmm I don't see any change